package.json

```diff
@@ -19,16 +19,16 @@
     "npm": ">=7.20.x <8.14"
   },
   "dependencies": {
-    "@bigcommerce/big-design": "^0.32.1",
+    "@bigcommerce/big-design": "^0.33.0",
     "@bigcommerce/big-design-theme": "^0.16.0",
-    "firebase": "^9.9.2",
+    "firebase": "^9.9.3",
     "jsonwebtoken": "^8.5.1",
     "mysql": "^2.18.1",
     "next": "^12.2.5",
```

Comment:
Maybe we should make a ticket to bump us to Next 13! Next.js Conf earlier this week has me like: 🤤😍